packages/terra-core-docs/src/terra-dev-site/doc/list/example/InstructionTextList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import List, { Item } from 'terra-list/lib/index';
+import { Placeholder } from '@cerner/terra-docs';
+import classNames from 'classnames/bind';
+import styles from './ListDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const InstructionTextList = () => (
+  <List instructionText="Here are some instructions for using this list.">
```

Comment:
Two different ways of providing instructions, as I was finding they don't both work in all browser-screen reader combinations. I wonder if it would make more sense to just call these props `ariaDescribedBy` and `ariaDescription`, since they are just passed directly through.

Reply:
That's a more intuitive name. +1!